docs/source/data.rst

```diff
@@ -22,6 +22,12 @@ Generic Interfaces
   :members:
   :special-members: __getitem__
 
+`PersistentDataset`
+~~~~~~~~~~~~~~
```

Comment:
The "~~~" should exactly match the length of `PersistentDataset`.

Reply:
Could you please help update this ~~~? Otherwise, the Document system will raise an error.
Thanks.